README.md

```diff
@@ -1,2 +1,63 @@
 # fsr
-FSR code used for dance pads
+A complete software package for FSR dance pads.
+
+## Features
+- React web UI to customize sensitivity 
+- Profiles & persistence
+- Light support
+
+## Screenshots
+<img src="./img/fsr2.gif">
+
+<img src="./img/fsr1.gif">
+
+
+## Requirements
+- A [Teensy](https://www.pjrc.com/store/index.html)
```

Comment:
Teensy or Arduino works.

The code is written to automagically support both (uses native keyboard library for Arduino, and Joystick library for teensy)